src/lifx/packets/setMultiZoneEffect.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+var _require = require('../../lifx'),
+    constants = _require.constants,
+    utils = _require.utils;
+
+var Packet = {
+  size: 59
+};
+
+/**
+ * Converts packet specific data from a buffer to an object
+ * @param  {Buffer} buf Buffer containing only packet specific data no header
+ * @return {Object}     Information contained in packet
+ */
+Packet.toObject = function (buf) {
+  var obj = {};
+  var offset = 0;
+
+  if (buf.length !== this.size) {
+    throw new Error('Invalid length given for setMultiZoneEffect LIFX packet');
+  }
+
+  obj.instanceid = buf.readUInt32LE(offset);
+  offset += 4;
+
+  obj.effect_type = buf.readUInt8(offset);
+  offset += 1;
+
+  // skip reserved field UInt16LE
+  offset += 2;
+
+  obj.speed = buf.readUInt32LE(offset);
+  offset += 4;
+
+  // Int64 is not supported. Not giving the value makes the effect infinite
+  //obj.duration = utils.readUInt64LE(buf, offset);
+  //obj.duration = new Int64(buf.readUInt32LE(offset), buf.readUInt32LE(offset + 4));
```

Comment:
Could something like this help: https://www.npmjs.com/package/node-int64 ?

Reply:
I tried with in64 implementations on my local copy but couldn't get it to work. Hence why I left that out as nodejs is about to add in64 support in the core at some point.